src/main/webapp/resources/js/pages/analysis/components/AnalysisSamples.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useContext, useEffect, useState } from "react";
+import { Card, Row, Icon, Button } from "antd";
+import { AnalysisDetailsContext } from '../../../contexts/AnalysisDetailsContext';
+import { getI18N } from "../../../utilities/i18n-utilties";
+
+import {
+    getAnalysisInputFiles,
+    getForwardFile,
+    getReverseFile
+} from "../../../apis/analysis/analysis";
+
+export default function AnalysisSamples() {
+  const { analysisDetailsContext, loadAnalysisSamples } = useContext(AnalysisDetailsContext);
+
+  useEffect(() => {
+    loadAnalysisSamples();
+  }, []);
+
+  const downloadReferenceFile = () => {
+    if (analysisDetailsContext.referenceFile.identifier !== undefined)
+    {
+        window.open(`${window.TL.BASE_URL}referenceFiles/download/${analysisDetailsContext.referenceFile.identifier}`, '_blank');
+    }
+  };
+
+  const renderReferenceFile = () => {
+    const referenceFile = [];
+
+    if(analysisDetailsContext.referenceFile.length == 0)
+    {
+        return null;
+    }
+    else {
+
+        referenceFile.push(
+            <h4 key="reference-file-0-heading" style={{fontWeight: "bold"}}>{getI18N("analysis.tab.content.samples.reference-file")}</h4>,
+            <Row key="row-filename">
+                <span key="reference-file-0-file">{analysisDetailsContext.referenceFile.label}</span>
+                <Button
+                    key="reference-file-0-download-button"
+                    className="pull-right"
```

Comment:
This is a bootstrap class, let's talk about what your trying to do.  Probably should use [`flexbox` ](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)for this

Reply:
Fixed in 239318b0bb57f7ffc2020d24cf154c055cf22b72